glue.tf

```diff
@@ -24,3 +24,14 @@ resource "aws_glue_catalog_database" "apiary_system_glue_database" {
   name         = "${local.gluedb_prefix}${var.system_schema_name}"
   description  = "Managed by Apiary terraform"
 }
+
+resource "null_resource" "automatic_glue_stats_collector_script" {
+  count        = var.disable_glue_db_init ? 1 : 0
```

Comment:
What does disable glue db init have to do with stats collection?

Reply:
Yeah, it is not really clear, when you disable glue init from HMS readwrite, that means all the Glue new stuff + lake formation permissions starts... but I will use enable_glue variable better, thank you